lib/ansible/plugins/connection/paramiko_ssh.py

```diff
@@ -324,7 +324,7 @@ def _connect_uncached(self):
                 key_filename = os.path.expanduser(self._play_context.private_key_file)
 
             ssh.connect(
-                self._play_context.remote_addr,
+                self._play_context.remote_addr.lower(),
```

Comment:
Do we need the same for the other connection plugins?

Reply:
I don't think so... this appears to be a paramiko bug (or at any rate for our purposes it is paramiko-specific), so we would only have to change other connections if they use paramiko for transport, but implement it directly rather than handing off to paramiko_ssh.